bindata/network/multus/002-rbac.yaml

```diff
@@ -52,14 +52,28 @@ metadata:
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
```

Comment:
er, I'm confused. This is the role assigned to the service account used by the multus binary itself, right? Then I doubt you can change this. I believe that adding the new Role below you've done what you need.

Reply:
Nice catch, this indeed should not have been changed, you're right -- all it needs is the additional Role.